### ISSUE TITLE
[Firefox 89][Proton][Custom CSS 3.2.4] icons and context menu

### ISSUE THREAD

**Issue by gabri25ele** [Aris-t2/CustomCSSforFx#374]
1) ehi where are the icons?

![ico](https://user-images.githubusercontent.com/33927879/121003825-3c8ba400-c78e-11eb-93b6-5df3d50b7d7a.png)

is it not enough to enable colorization?

2) context menu

is that normal?

![context](https://user-images.githubusercontent.com/33927879/121003901-54fbbe80-c78e-11eb-9a72-8deaaac39983.png)

I didn't touch anything in the userchrome file

**Comment by Aris-t2** [comment]
This project does not offer icons inside appmenu (yet). It is planned for a future update.
In another thread there was a link to a project, that already offers those icons.

**Comment by motfis** [comment]
I take part of code from https://github.com/black7375/Firefox-UI-Fix

[iconize_menu.zip](https://github.com/Aris-t2/CustomCSSforFx/files/6611377/iconize_menu.zip)
<img width="267" alt="Capture2" src="https://user-images.githubusercontent.com/8572695/121073862-60250d80-c7d3-11eb-96e4-935d0b29667c.PNG">

**Comment by gabri25ele** [comment]
> 
> 
> I take part of code from https://github.com/black7375/Firefox-UI-Fix

thanks god!

**Comment by Pizzapops** [comment]
**motfis** 
I found adding my **CustomJSforFx**  js scripts (16) to **black7375**'s Chrome folder, makes it even nicer. I am getting frustrated with **FxProton**. **Vivaldi** is now my default Browser with **FxQ-esr** & **Waterfox** as secondary.

**Comment by Aris-t2** [comment]
I switched to Vivaldi after Fx 57 release ;-)

**Comment by motfis** [comment]
> **motfis**
> I found adding my **CustomJSforFx** js scripts (16) to **black7375**'s Chrome folder, makes it even nicer. I am getting frustrated with **FxProton**. **Vivaldi** is now my default Browser with **FxQ-esr** & **Waterfox** as secondary.

I just use part of the code, to show the icon **only**. My main tweaks still use the one from this git (customCSS4Fx) - more customize-able. It can be even more colorful by modifying slightly the _appbutton_popup_icons_colorized.css_.


> I switched to Vivaldi after Fx 57 release ;-)

Interesting. For me Vivaldi seems slower than Fx, so I use Vivaldi as my 2nd browser.

**Comment by gabri25ele** [comment]
> 
> 
> I switched to Vivaldi after Fx 57 release ;-)

Vivaldi has flaws and above all very little direct customization of the interface

one day you will come back!

**Comment by gabri25ele** [comment]
> I just use part of the code, to show the icon **only**. My main tweaks still use the one from this git (customCSS4Fx) - more customize-able. It can be even more colorful by modifying slightly the _appbutton_popup_icons_colorized.css_.

tell me more, and more.

**Comment by Roadfox-Master** [comment]
Hello, I'm new here, but I read this page since many months. I found a bug in css/generalui/context_bfrsb_labels_with_icons.css in line 94 and 95. Original:
-moz-margin-start: 0px !important;
-moz-margin-end: 0px !important;
but icons in context menu are not centered in the column. I set it to 14px and icons back, forward, ... are now centerd
![Context](https://user-images.githubusercontent.com/85829875/121806652-01d3b100-cc51-11eb-9c3e-e92ca531554c.jpg)
